Bottom: fd1d725d2676eb9abe694b4034a8afc2edd36081
Top:    2fab5dd88e216f1c46e2e38d7a17d74218db5b12
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-09 12:59:51 +0100

Refresh of fix-layer-on-off-rendering

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ecc02ff..bf0c723 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1540,7 +1540,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
     l = global_view_2d ? LAYER_ON_STACK (i) : LAYER_PTR (orderi);
     group = GetLayerGroupNumberByNumber (global_view_2d ? LayerStack[i] : orderi);
 
-    if (/*l->On && */!do_group[group]) {
+    if (l->On && !do_group[group]) {
       do_group[group] = 1;
       drawn_groups[ngroups++] = group;
     }
